ts/change-notetype/Alert.svelte

```diff
@@ -25,11 +25,27 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         ? tr.changeNotetypeExpand()
         : tr.changeNotetypeCollapse();
     $: icon = collapsed ? plusIcon : minusIcon;
+
+    function onKeyDown(event: KeyboardEvent) {
+        switch (event.code) {
+            case "Enter":
+            case "Space":
+                collapsed = !collapsed;
+                break;
+        }
+    }
```

Comment:
It might be worth factoring this out into a helper function so we can use it like onEnterOrSpace(() => (collapsed = !collapsed))

Reply:
You previously mentioned something similar to that [in the original #2564 issue](https://github.com/ankitects/anki/issues/2564#issuecomment-1634072182). Given how common those buttons being synonymous are in the the ARIA APG, I would agree that that would be good as a universal function. Also, it would be helpful for consistency in "select the thing I am focusing on" (I might prefer to call it something like onSelect() on onAriaSelect()). Though identifying what in the code base that would actually work on I think would take a bit and be best suited to a different issue or PR. I'll look into it.